### PR TITLE
day-workout

### PR DESCRIPTION
**Pull request by Zaki5581**

### CODE REVIEW

File: src/style/StyledMain.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  h2 {
+    display: flex;
+    justify-content: center;
+  }
+  select {
+    width: 11rem;
+    height: 1.5rem;
+    margin: auto;
+  }
+  .workout-plan {
```
Comment (JessicaLoers) [inline src/style/StyledMain.js:18]: I recommend that to not work with classes, but with styled compoments for these elements.

**Review by JessicaLoers — COMMENTED** [review]
Hi @Zaki5581 ✌️
Good work. I have some suggestions and comments you could think about.

File: src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    setPlans([...plans, {name: newPlan, id: nanoid()}]);
+  }
+  // workouts store
+  useEffect(() => {
+    localStorage.setItem('my_plan', JSON.stringify(weeks));
+  }, [weeks]);
+  //setting the sets
+  function updateSets(id, value) {
+    setPlans([...plans.slice(0, id), {...plans[id], sets: value}, ...plans.slice(id + 1)]);
+  }
```
Comment (macfrei) [inline src/components/Main.js:28]: Please rename these to `index` as it is the `index`:
```suggestion
  function updateSets(index, value) {
    setPlans([...plans.slice(0, index), {...plans[index], sets: value}, ...plans.slice(index + 1)]);
  }
```

File: src/components/Main.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  function saveWeek() {
+    const newSet = {exercise: plans, bodyPart: selectedBodypart};
+    setWeeks([...weeks, newSet]);
+    setPlans([]);
+    setSelectedBodypart('');
+  }
+  //delete day-plan
+  function deletMyPlan(id) {
+    setWeeks([...weeks.slice(0, id), ...weeks.slice(id + 1)]);
+  }
```
Comment (macfrei) [inline src/components/Main.js:39]: Please rename this to `index` as this is an `index`:
```suggestion
  function deletMyPlan(index) {
    setWeeks([...weeks.slice(0, index), ...weeks.slice(index + 1)]);
  }
```

**Review by macfrei — APPROVED** [review]
Looks good!

**Pull request merged**